openerp/service/server.py

```diff
@@ -232,7 +232,12 @@ def close_socket(self, sock):
         sock.close()
 
 class ThreadedServer(CommonServer):
+    initial_threads = []
+    
     def __init__(self, app):
+        for t in threading.enumerate():
+            self.initial_threads.append(t)
```

Comment:
Further down the code:
"and thread **not** in self.initial_threads"
so it will only skip threads in the list, which should only contain the main thread, and maybe the pydev thread since, like you said, Odoo has not spawned anything yet.
Anyway, thanks for considering. I understand I should find a way to daemonify the pydev thread instead.